roles/falcon_install/tasks/preinstall.yml

```diff
@@ -54,13 +54,13 @@
 
 - name: "CrowdStrike Falcon | Endpoint Operating System Detected Is Debian"
   ansible.builtin.set_fact:
-    falcon_os_version: "9/10/11"
+    falcon_os_version: "*{{ falcon_os_version }}*"
   when:
     - ansible_distribution == "Debian"
 
 - name: "CrowdStrike Falcon | Endpoint Operating System Detected Is Ubuntu"
   ansible.builtin.set_fact:
-    falcon_os_version: "16/18/20"
+    falcon_os_version: "*{{ falcon_os_version }}*"
```

Comment:
Heads up in puppet it returns `16.04`, `18.04` etc. The api only expects `16`, `18`, etc.

This is what I did in Puppet to handle that `os_release_major.split('.')[0]}` Split on period and grab first element.

Reply:
I'll double check 16/18 but 20.x does report back 20.

Reply:
Double checked and it looks good